src/js/view/view.js

```diff
@@ -805,6 +814,8 @@ define([
             _showing = true;
             utils.removeClass(_playerElement, 'jw-flag-user-inactive');
 
+            _controlbar.checkCompactMode(_videoLayer.clientWidth);
+
```

Comment:
Turn on "Show paint rectangles" in chrome or do some display profiling to see if mouse movement thrashes performance. That's my concern with the previous comment.